examples/flutter_gallery/lib/demo/material/chip_demo.dart

```diff
@@ -28,7 +28,8 @@ class _ChipDemoState extends State<ChipDemo> {
       ),
       const Chip(
         avatar: const CircleAvatar(child: const Text('B')),
-        label: const Text('Blueberry')
+        label: const Text('Blueberry'),
+        border: const BorderSide(color: Colors.redAccent, width: 4.0),
```

Comment:
did you mean to leave this border side? I don't think I've seen chips with a border like this in the material spec.

Reply:
Yeah, I wanted to leave it as an example, but you're right that it's not in the spec.  Removed.